p2p/net/handshake.go

```diff
@@ -74,6 +74,8 @@ func GenerateHandshakeRequestData(localPublicKey crypto.PublicKey, localPrivateK
 	data.Hmac = hm.Sum(nil)
 	data.Sign = ""
 
+	data.Port = uint32(port)
```

Comment:
uint16

Reply:
`data` is a protobuf struct. only receives (u)int32/64